src/app/learn/get-started/prereqs/initial-setup/index.html

```diff
@@ -17,11 +17,22 @@
     Install the latest LTS release of <a href="https://nodejs.org/en/download/releases/">Node.js</a>. We require at least version 6. To check your version, run <stache-code>node –v</stache-code> from the command line.
   </p>
   <p>
-    For Windows, download the <stache-code>msi</stache-code> version of the latest LTS release.
+    For Mac OS X and Linux, we recommend that you use <a href="https://github.com/creationix/nvm">Node Version Manager (nvm)</a> to wrap your NodeJS installation so that it installs in your user directory and avoids permissions-related issues.
   </p>
   <p>
-    For Mac OS X and Linux, we recommend that you use <a href="https://github.com/creationix/nvm">Node Version Manager (nvm)</a> to wrap your NodeJS installation so that it installs in your user directory and avoids permissions-related issues.
+    For Windows, download the <stache-code>msi</stache-code> version of the latest LTS release. Windows machines also require two additional steps:
   </p>
+  <ul>
+    <li>
+      To install the node-sass requirement as described in <a href="https://github.com/sass/node-sass#install">the official node-sass documentation</a>, run <stache-code>npm install node-sass</stache-code>.
+    </li>
+    <li>
+      To install the required tools and configurations using <a href="https://github.com/felixrieseberg/windows-build-tools">Microsoft's windows-build-tools</a>, run <stache-code>npm install --global --production windows-build-tools</stache-code> from an elevated PowerShell or CMD.exe (run as administrator).
+    </li>
+  </ul>
+<!--  <p>
```

Comment:
Is there a reason that we are leaving in this commented block?

Reply:
Just as a reminder to make additional doc updates. I'd had additional docs in this PR but pulled them after talking to Bobby and James Henderson. We are going to expand the docs to include `nvm` recommendations for Windows, and I'll clear out that commented block when I update the docs again. Hopefully next week.

Reply:
Sounds good.